fastchat/serve/cli.py

```diff
@@ -6,11 +6,115 @@
 import time
 
 import torch
-from transformers import AutoTokenizer, AutoModelForCausalLM, LlamaTokenizer
+from transformers import AutoTokenizer, AutoModel, AutoModelForCausalLM, LlamaTokenizer
+from transformers.generation.logits_process import (
+    LogitsProcessorList,
+    TopKLogitsWarper,
+    TopPLogitsWarper
+)
 
 from fastchat.conversation import conv_templates, SeparatorStyle
 
 
+def prepare_inputs(input_ids, output_ids, past_key_values, device):
```

Comment:
Move these ChatGLM-specific functions to a new file `fastchat/serve/chatglm.py`?